src/lumigo_tracer/user_utils.py

```diff
@@ -11,3 +13,38 @@ def report_error(msg: str):
     else:
         message_with_request_id = f"RequestId: {SpansContainer.get_span().function_span.get('id')} {message_with_initials}"
         print(message_with_request_id)
+
+
+def add_execution_tag(key: str, value: str, should_log_errors: bool = True) -> None:
+    """
+    The maximum number of tags is 50.
```

Comment:
Ask "Alex" to phrase something better for you.
I would change it to something like "Use this function to tag the current invocation with dynamic information, that you can query later in the platform."
Be friendly! Haha

Reply:
You're right, at first I didn't understand exactly what you meant :)